README.md

```diff
@@ -14,7 +14,7 @@ Pieman is a core component of [CusDeb](https://cusdeb.com).
 
 ## Features
 
-* The images can be based on [Alpine](https://alpinelinux.org) and Debian-based distributions such as [Kali](https://kali.org), [Raspbian](https://www.raspberrypi.org/downloads/raspbian/) and [Ubuntu](https://ubuntu.com/).
+* The images can be based on [Alpine](https://alpinelinux.org) and Debian-based distributions such as [Kali](https://kali.org), [Raspberry Pi OS](https://www.raspberrypi.org/software/operating-systems/#raspberry-pi-os-32-bit) and [Ubuntu](https://ubuntu.com/).
```

Comment:
1. Since we edit the address, let's remove `www.`
2. Let's add `(formerly Raspbian)` after `[Raspberry Pi OS](...)`